aggregator_type: ENDTIME
aggregation_time_topic: /aggregation_time
pointcloud_topic: /hvlp/velodyne_points
aggregate_topic: /hvlp/aggregates
odometry_topic: /odometry/ground_truth
log_directory: /.ros/log/beam_slam/lidar_aggregation
dynamic_extrinsics: false
clear_queue_on_update: false
max_aggregation_time_seconds: 5
